Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM opensuse/tumbleweed
+FROM opensuse/leap:15.3
```

Comment:
```suggestion
FROM registry.opensuse.org/opensuse/leap:15.3
```

Reply:
I looked on registry.opensuse.org, but didn't put it in the Dockerfile :man_facepalming: